### DISCUSSION TITLE
Elastic-fleet - many broken Kibana links

### DISCUSSION THREAD

**Discussion by CharlieMK** [Security-Onion-Solutions/securityonion#11180] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

1. When I navigate to Elastic fleet => click any agent => click 'view more agent metrics' I get a Kibana dashboard that errors out:
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/63464653/f9c02b68-2153-403d-9b8e-254ba52d1bd6)

2. When navigating to Kibana => Agent info dashboard => click any of the links in the navigation menu on the left (see screenshot below): I get a 404. It seems 'kibana' is not in the path of the url. 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/63464653/1464ad97-3c52-4bc5-8f57-b801c627777e)

I suppose these are two separate issues. Any way to solve these or are they already identified as issues?

(I have followed the solution to the elastic-defend integration as explained in the answer of https://github.com/Security-Onion-Solutions/securityonion/discussions/11084à

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by defensivedepth** [comment]
This is a known issue, and will be fixed in the next release:

https://github.com/Security-Onion-Solutions/securityonion/issues/11018
